### PR TITLE
Feat/avatar component

### PR DESCRIPTION
**Pull request by haydencleary**
First step towards new `User` component

### CODE REVIEW

File: components/ds/avatar/avatar.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      </div>
+    );
+  },
+};
+
+export const Radius: Story = {
+  render: args => {
+    return (
+      <div className="flex w-full items-center gap-2">
+        <Avatar {...defaultProps} {...args} radius="full" />
```
Comment (alexbeno) [inline components/ds/avatar/avatar.stories.tsx:50]: Can you add the same variant as the thumbnail component ? the size and variant as the same as figma ?

**Review by alexbeno — APPROVED** [review]
Nice one

File: components/ds/avatar-group/avatar-group.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <div className={cn(avatarGroupVariants({ ...props, size: avatarProps?.size }), className)}>
+      {avatars.map((avatar, index) => (
+        <Avatar
+          key={index}
+          {...avatar}
+          {...avatarProps}
+          className="transition-transform hover:-translate-x-1/4 hover:last:translate-x-0"
+        />
+      ))}
+    </div>
```
Comment (alexbeno) [inline components/ds/avatar-group/avatar-group.tsx:18]: Why did you not use this : 

https://nextui.org/docs/components/avatar#avatar-group

Comment (haydencleary) [reply components/ds/avatar-group/avatar-group.tsx:18]: I did at first but it made things more complicated than necessary. The only thing I found it useful for was the animation so I did it myself.

**Comment by haydencleary** [comment]
Just realised a component I deleted was still in use, going to fix this now.

**Comment by haydencleary** [comment]
> Just realised a component I deleted was still in use, going to fix this now.

Fixed

File: components/ds/avatar-group/avatar-group.tsx (hunk 2 above)
Comment (alexbeno) [reply components/ds/avatar-group/avatar-group.tsx:18]: ok in this case can you add a props to select how many avatar we want to show and if exceed we add a +N ( like the AvailableConversion component ? ) or we keep this for later ?

Comment (haydencleary) [reply components/ds/avatar-group/avatar-group.tsx:18]: We can handle this when we get there

**Pull request merged**